### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role.git`](https://github.com/cisagov/skeleton-ansible-role.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ansible-role-clamav.git ansible-role-clamav
    cd ansible-role-clamav
    git remote add skeleton https://github.com/cisagov/skeleton-ansible-role.git
    git remote set-url --push skeleton no_push
    git switch develop
    git switch --create lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/dependabot.yml .github/workflows/build.yml meta/main.yml molecule/default/molecule-with-systemd.yml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Confirm each item in the "Pre-approval checklist" below.

1. Remove any of the checklist items that do not apply.

1. Ensure every remaining checkbox has been checked.

1. Mark this draft pull request "Ready for review".

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] ✌️ The conflicts in this pull request have been resolved.
- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

---

> [!NOTE]
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Comment by jsf9k** [comment]
The only ways I can think of to fix this recurrent and stubborn method of failure are:
- Create a public ClamAV mirror and update this role (and therefore all our AMIs) to use it
- Create an autoscaling Kubernetes deployment for self-hosted GH Actions runners

The latter would be based on [GitHub's Actions Runner Controller](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller), leveraging something like [CloudPosse's Terraform module](https://github.com/cloudposse/terraform-aws-components/tree/main/modules/eks/actions-runner-controller) for standing up the EKS cluster.  I think this is the best solution, since self-hosted runners are something that would be useful for other repos in our org as well.

What do you think, @cisagov/team-ois?

**Comment by dav3r** [comment]
I am on board with us spinning up some GH runners- I agree that this will be useful to us in other situations.  I assume we are talking about deploying this EKS cluster in one of our AWS accounts?  So not exactly "self-hosted", but non-GH-hosted.

**Comment by jsf9k** [comment]
> I am on board with us spinning up some GH runners- I agree that this will be useful to us in other situations. I assume we are talking about deploying this EKS cluster in one of our AWS accounts? So not exactly "self-hosted", but non-GH-hosted.

Yes, in one of our AWS accounts.

**Comment by jsf9k** [comment]
@cisagov/team-ois - Any ideas why this PR would repeatedly and consistently fail the `build/test` checks while cisagov/ansible-role-clamav#72 and cisagov/ansible-role-clamav#73, for instance, passed with no issue?  I think this may be an important clue.

**Comment by jsf9k** [comment]
I want to merge this one despite the bizarrely failing tests, so we don't get too far behind.  Does that sound OK with you, @cisagov/team-ois?

**Comment by dav3r** [comment]
> I want to merge this one despite the bizarrely failing tests, so we don't get too far behind. Does that sound OK with you, @cisagov/team-ois?

If we have no better options, then I'm fine with doing that.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,31 +5,42 @@
...
-      - dependency-name: "ansible"
-      - dependency-name: "ansible-lint"
+      - dependency-name: ansible
+      - dependency-name: ansible-lint
+=======
+    # ignore:
+    #   # Managed by cisagov/skeleton-ansible-role
+    #   - dependency-name: ansible
+    #   - dependency-name: ansible-lint
+>>>>>>> 166674527f743553dfae7b193a89b18f6460ef45
```
Comment (dav3r) [inline .github/dependabot.yml]: @jsf9k - This is definitely a problem - merge markers in `dependabot.yml`.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -115,9 +158,17 @@ jobs:
           - ubuntu
           - ubuntu_additional_flags
           - ubuntu_modify_config
-    runs-on: ubuntu-latest
+=======
+          - default
+          - systemd_enabled
+>>>>>>> 166674527f743553dfae7b193a89b18f6460ef45
```
Comment (dav3r) [inline .github/workflows/build.yml]: More merge markers in this file as well.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -105,15 +111,19 @@ repos:
     hooks:
       - id: bandit
         # Bandit complains about the use of assert() in tests
+<<<<<<< HEAD
         exclude: molecule/(additional_flags|default|modify_config)/tests
+=======
+        exclude: molecule/(default|systemd_enabled)/tests
+>>>>>>> 166674527f743553dfae7b193a89b18f6460ef45
```
Comment (dav3r) [inline .pre-commit-config.yaml]: More merge markers here.

File: meta/main.yml [hunk 4]
```diff
@@ -35,4 +40,9 @@ galaxy_info:
       versions:
         - focal
         - jammy
+<<<<<<< HEAD
   role_name: clamav
+=======
+  role_name: skeleton
+>>>>>>> 166674527f743553dfae7b193a89b18f6460ef45
+  standalone: true
```
Comment (dav3r) [inline meta/main.yml]: Merge markers here too.

File: molecule/default/molecule-with-systemd.yml [hunk 5]
```diff
@@ -10,86 +10,98 @@ dependency:
...
+  - cgroupns_mode: host
+    command: /lib/systemd/systemd
+    image: geerlingguy/docker-amazonlinux2023-ansible:latest
+    name: amazonlinux2023-systemd
+    platform: amd64
+    pre_build_image: true
+    privileged: true
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+>>>>>>> 166674527f743553dfae7b193a89b18f6460ef45
```
Comment (dav3r) [inline molecule/default/molecule-with-systemd.yml]: Merge markers here too.

File: .github/dependabot.yml (hunk 1 above)
Comment (jsf9k) [reply .github/dependabot.yml]: Something got hosed when I rebased.  I just redid the upstream merge and force pushed, so please re-review.

**Review by dav3r — APPROVED** [review]
LGTM

**Review by jasonodoom — APPROVED** [review]

**Pull request merged**